src/utils/api/client.ts

```diff
@@ -54,3 +54,18 @@ export const apiPaymentEcommerceClientWithRetry = createEcommerceClient({
     async (r: Response): Promise<boolean> => r.status > 499
   ),
 });
+
+/**
+ * Api client for ecommerce API calculate fee with retry execution
+ */
+export const apiPaymentEcommerceClientWithRetryV2 = createEcommerceClientV2({
```

Comment:
Why create a brand new client with the second version of EccommerceClient and use it for "fees"?
Is it correct to assume that only Fees will use it and not in the other points of the webapp? Cause, if this assumption is not true, maybe it's better to modify the existing one `apiPaymentEcommerceClientWithRetry`

Reply:
`apiPaymentEcommerceClientWithRetry` is actually used for the following api calls:
- requestTransactionAuthorization
- createSession

eCommerce side only modified api will be included in the next version (v2 by the moment) and this is the case for the calculateFees one (for handle calculate fees with multiple payment notices).
So by the moment it's mandatory to leave both clients, one per v1 api's, one for v2 ones